projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts

```diff
@@ -78,21 +78,25 @@ export class ResourceViewComponent implements OnInit, OnChanges, OnDestroy {
         private _valueOperationEventService: ValueOperationEventService) { }
 
     ngOnInit() {
-        // subscribe to the event bus and listen for the ValueAdded event to be emitted
-        // when a ValueAdded event is emitted, get the resource again to display the newly created value
+        // subscribe to the ValueOperationEventService and listen for an event to be emitted
         this.valueOperationEventSubscription = this._valueOperationEventService.on(
-            Events.ValueAdded, (newValue: ReadValue) => this.updateResource(newValue, false));
+            Events.ValueAdded, (newValue: EventValues) => this.updateResource(newValue.currentValue, 'create'));
 
         this.valueOperationEventSubscription = this._valueOperationEventService.on(
-            Events.ValueDeleted, (deletedValue: DeleteValue) => this.updateResource(deletedValue, true));
+            Events.ValueUpdated, (updatedValue: EventValues) =>
+                this.updateResource(updatedValue.currentValue, 'update', updatedValue.newValue));
+
+        this.valueOperationEventSubscription = this._valueOperationEventService.on(
+            Events.ValueDeleted, (deletedValue: EventValues) => this.updateResource(deletedValue.currentValue, 'delete'));
+
     }
 
     ngOnChanges() {
         this.getResource(this.iri);
     }
 
     ngOnDestroy() {
-        // unsubscribe from the event bus when component is destroyed
+        // unsubscribe from the ValueOperationEventService when component is destroyed
         if (this.valueOperationEventSubscription !== undefined) {
```

Comment:
this should the rather be an array of subscriptions

Reply:
done in dda33f8